_data/projects/cyclonedx-dotnet.yml

```
@@ -0,0 +1,11 @@
+name: CycloneDX .NET Implementation
+desc: Creates CycloneDX Software Bill-of-Materials (SBOM) from .NET Projects
+site: https://cyclonedx.org/
+tags:
+  - sbom
+  - .net
+  - .netcore
+  - cyclonedx
+upforgrabs:
+  name: good first issue
+  link: https://github.com/CycloneDX/cyclonedx-dotnet/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22
```

A bit of cleanup to enable the GitHub integration:

```suggestion
  link: https://github.com/CycloneDX/cyclonedx-dotnet/labels/good%20first%20issue
```